gorgon.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |s|
 
   s.add_development_dependency "rake"
   s.add_development_dependency "test-unit"
-  s.add_development_dependency "rspec", '~>2.11.0'
+  s.add_development_dependency "rspec", '~>3'
```

Comment:
Thanks for this pull request @waseem! We probably want to use pessimistic version constraint (aka '~> 3.5.0'), so it's easier to upgrade patch versions of RSpec without upgrading to an RSpec version that breaks our tests.